bedrock/firefox/templates/firefox/new/scene1_windows.html

```diff
@@ -0,0 +1,49 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/new/quantum" %}
+
+{% extends "firefox/new/scene1.html" %}
+
+{% block page_title_prefix %} {{_('Download Mozilla Firefox')}} {% endblock %}
+{%- block page_title -%} {{_('for Windows')}} {%- endblock -%}
+{%- block page_title_suffix -%} {{_(' — Free Web Browser')}} {%- endblock -%}
+
+{% block page_desc %}
+  {{ _('Download Mozilla Firefox for Windows, a free Web browser. Firefox is created by a global non-profit dedicated to putting individuals in control online. Get Firefox for Windows today!') }}
+{% endblock %}
+
+{#- This will appear as <meta property="og:title"> which can be used for social share -#}
+{% block page_og_title %}{{_('Download the fastest Firefox for Windows ever')}}{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{_('Faster page loading, less memory usage and packed with features, the new Firefox for Windows is here.')}}{% endblock %}
+
+{% block header_content %}
+<header>
+  <h1>{{ _('Firefox fights for you') }}</h1>
+  <h2>{{ _('on Windows.') }}</h2>
```

Comment:
The original page headline was two distinct statements: "The new Firefox" and "Fast for good." so it was done with two headings. But this version is one statement that should read as one statement, but they're emulating the visual styling of the old headline... semantically this should be a single heading with a `span` for the visual formatting but I don't know if it's worth the added effort. Your call.